Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -2,8 +2,15 @@ import UIKit
 
 public protocol GestureManagerDelegate: AnyObject {
 
-    /// Informs the delegate that a gesture haas begun. Could be used to cancel camera tracking.
-    func gestureBegan(for gestureType: GestureType)
+    /// Informs the delegate that a gesture has begun.
+    func gestureManager(_ gestureManager: GestureManager, didBegin gestureType: GestureType)
+
+    /// Informs the delegate that a gesture has ended and whether there will be additional animations after the gesture
+    /// has completed. Does not indicate whether gesture-based animations have completed.
```

Comment:
I would remove "Does not indicate whether gesture-based animations have completed."

Reply:
I think the rest is sufficiently clear